### ISSUE TITLE
status?

### ISSUE THREAD

**Issue by MACscr** [adapt0/smartplug#1]
I know its a work in progress, but what state is the project in? Do you have it working? Is it just on and off or does it do power monitoring as well with the new firmware?

**Comment by adapt0** [comment]
Yeah, basics are there. Relay is toggleable via a web interface. Power monitoring is hooked up as a proof of concept, but requires some further tweaking + calibration before being useful.

After managing to get the remote firmware hijacking running I got distracted by work...

**Comment by adapt0** [comment]
<img width="1092" alt="smartplug" src="https://user-images.githubusercontent.com/2055702/45579526-2a7ab200-b83d-11e8-8c1f-946534d674dc.png">

Voltage seems reasonable. Wattage values are pretty random.

**Comment by MACscr** [comment]
Just revisiting this 2 years later, but still happy with its current state? Are you simply using that custom web interface or interacting with it mainly though another smart home type panel?

**Comment by adapt0** [comment]
Honestly these have ended up in the bottom of my drawer. It was one of those projects where after overcoming the main challenges it turned into a project too similar to work.

That said, I keep meaning to come back to it, as it would be useful to use these for some power cycle testing of other devices.
